test/verify/check-connection

```diff
@@ -172,33 +172,32 @@ class TestConnection(MachineCase):
         m = self.machine
 
         if not m.image in ["rhel-7-5"]:
-            self.assertIn("systemctl", m.execute("cat /run/cockpit/issue"))
-            self.assertNotIn("9090", m.execute("cat /run/cockpit/issue"))
+            self.assertIn("systemctl", m.execute("cat /run/cockpit/motd"))
+            self.assertNotIn("9090", m.execute("cat /run/cockpit/motd"))
```

Comment:
Can you please change the tests to check "/etc/motd.d/cockpit", so that this also covers the correct packaging/installation of that symlink?